_posts/2021-02-01-hhvm-4.95.markdown

```diff
@@ -0,0 +1,41 @@
+---
+title: "HHVM 4.95"
+layout: post
+author: jjergus
+category: blog
+---
+
+HHVM 4.95 is released! This release marks the end of support for 4.87;
+HHVM 4.88&ndash;4.94 remain supported, as do the 4.56 and 4.80 LTS releases.
+
+(Note: HHVM 4.89 and 4.90 were skipped.)
+
+# Highlights
+
+- Fixed an issue where overriding a type constant from a parent class/interface
+  could cause a runtime error even if the code passed the typechecker
+  ([example](https://github.com/facebook/hhvm/commit/206760e532d7f58084d9f32904d35b6735f57e66#diff-980a17de858c05e56fd438cc18277928d6403d191640f53d25c0302af056b22d)).
+- Fixed various typechecker error messages incorrectly refering to a type
+  constant as "property".
+- The SOAP extension now supports proxy authentication (new options
+  `proxy_ssl_cert_path`, `proxy_ssl_key_path`, `proxy_ssl_ca_bundle`).
+
+# Breaking Changes
+
+- The builtin function `debug_backtrace()` now returns arguments (the `'args'`
+  key in the returned array) in a `vec` (previously `varray`).
+- Return type annotations were added to various
+  [`DateTime`](https://github.com/facebook/hhvm/commit/b31d8cc07fa084ce6a910128eeaae4e7edec36d1#diff-363445008aa8e1350233eb217353a35f808a91d29ba3fd382f24e2bb9244f80b)
+  methods (matching what these methods already returned). This may reveal
+  previously hidden typechecker errors.
+
+# Future Changes
+
+- Calling a function or method "dynamically" (e.g. calling a function/method
+  whose name is stored in a variable) will become more restricted in the future.
+  - See
+    [previous announcement](https://hhvm.com/blog/2020/11/30/hhvm-4.86.html#future-changes)
+    for more details.
+  - This release adds a new builtin function
+    `dynamic_meth_caller(string $cls_name, string $meth_name)`
+    that should be used to call *non-static* methods dynamically.
```

Comment:
I have a weak preference for 'instance' over 'non-static'.

Reply:
I'll mention both just to make it super clear.